apps/api/test/report-util.ts

```diff
@@ -67,3 +83,61 @@ export const getEducation = (options?: EducationOptions): NursingEducationDTO =>
     num_years: faker.helpers.arrayElement([1, 2, 3, 4]),
   };
 };
+
+export const HEALTH_AUTHORITIES = [
```

Comment:
We can get this data by using `Repository` or referencing `Service`

Reply:
ahh forgot about these